src/pages/search/CristinSearchPagination.tsx

```diff
@@ -25,6 +27,8 @@ export const CristinSearchPagination = ({ totalCount, page, rowsPerPage }: Crist
       onPageChange={(newPage) => updatePath(newPage.toString(), rowsPerPage.toString())}
       rowsPerPage={rowsPerPage}
       onRowsPerPageChange={(newRowsPerPage) => updatePath('1', newRowsPerPage.toString())}
-    />
+      showPaginationTop={true}>
```

Comment:
for bolske props holder det med navnet for å sette til true
```suggestion
      showPaginationTop>
```